client/src/ui/product/CommandPalette.vue

```diff
@@ -8,24 +8,44 @@
 
   const showDialog = ref(false);
 
-  const { nameToBindingKeys } = graph.value.shortcut;
+  const { platformBindings } = graph.value.shortcut;
+
+  const bindings = computed(() => {
+    const result: Record<string, string> = {};
+    const bindingNames = Object.entries(platformBindings);
+    for (const [name, bindingInfo] of bindingNames) {
+      result[name] = bindingInfo.binding;
+    }
+    return result;
+  });
+
+  const keyToNameMap: Record<string, string> = {
```

Comment:
Id specify that this is a user facing string and call this something like KEYBOARD_KEY_TO_USER_STRING

also I would sub out the type annotation for an `as const`